### PR TITLE
TI-2943 - Update Appurtenancies Schema

### PR DESCRIPTION
**Pull request by stewartm83**
Please review

### CODE REVIEW

**Comment by AnthonyThevenin** [comment]
Why Water and Sewer? I thought we said we were going to do only the Sewer because we don't have data for Water to test? And if I am right, how did you manage to test that migration for both Water and Sewer?

**Comment by stewartm83** [comment]
@AnthonyThevenin  Water changes are property box **header names** only, nothing on the actual database schema yet. It is just organizing the properties. 

I can remove it if you want because it does not have an effect on the database.

File: migrations/main/2018-11-26-update_sewer_water_appurtenances.sql [hunk 1]
```diff
@@ -0,0 +1,76 @@
+TRUNCATE TABLE "SewerAuxAppurtenance_AsBuilt";
+TRUNCATE TABLE "SewerAuxAppurtenance_Existing";
+ALTER TABLE "SewerAuxAppurtenance_AsBuilt" ADD COLUMN "Rotation" DOUBLE PRECISION;
```
Comment (bmharper) [inline migrations/main/2018-11-26-update_sewer_water_appurtenances.sql]: I'm a little confused by the migration code here:
1. It looks like it's been hand-tweaked, so that some columns are renamed. Other columns are dropped & recreated. Were some of them tweaked, and if so, why? Since you're truncating the tables anyway, I can't think of any advantage in renaming fields vs just letting the auto-generated code drop & recreate.
2. Changes to `SewerAuxAppurtenance_Existing` and `SewerAuxAppurtenance_AsBuilt` are intermingled. This makes it hard to follow.

**Comment by AnthonyThevenin** [comment]
Oh I see! If the changes for Water are part of the story then leave it there. I was confused because the story only mentions Sewer (TI-2943: Add Appurtenances to Main Schema for Sewer Module). I haven't been through all the e-mails attached to the ticket though!

**Comment by bmharper** [comment]
OK - we've chatted about the auto-generated SQL scripts. LGTM

File: migrations/main/2018-11-26-update_sewer_water_appurtenances.sql (hunk 1 above)
Comment (stewartm83) [reply migrations/main/2018-11-26-update_sewer_water_appurtenances.sql]: FYI, PostgreSQL error without tweak

_main: pq: column "Pressure_Rating" cannot be cast automatically to type double precision_
